src-java/testing/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceV2.java

```diff
@@ -211,4 +212,8 @@ BfdPropertiesPayload setLinkBfd(SwitchId srcSwId, Integer srcPort, SwitchId dstS
     HaFlowRerouteResult rerouteHaFlow(String haFlowId);
 
     HaFlowPingResult pingHaFlow(String haFlowId, HaFlowPingPayload payload);
+
+    List<HaFlowHistoryEntry> getHaFlowHistory(String flowId, Long timeFrom, Long timeTo);
+
+    List<HaFlowHistoryEntry> getHaFlowHistory(String flowId, Long timeFrom, Long timeTo, Integer maxCount);
```

Comment:
It seems like we can use only one version of the method with all params and then just use the default value null for required params when call this api from HaFlowHelper:
current implementation
   HaFlowHistory getHistory (String id, Long timeFrom = null, Long timeTo = null, Integer maxCount = null){
        return new HaFlowHistory (northboundV2.getHaFlowHistory(id, timeFrom, timeTo, maxCount))
    }